src/utilities.cc

```diff
@@ -173,38 +174,33 @@ void SetVersionString(Isolate* isolate) {
     Nan::Utf8String component_version(version_value);
     if (*component_name == nullptr || *component_version == nullptr) continue;
 
-    // Don't duplicate the Node.js version
     if (!strcmp("node", *component_name)) {
+      // Put the Node.js version on the first line, if we didn't already have it
       if (version_string == UNKNOWN_NODEVERSION_STRING) {
         version_string = "Node.js version: v";
         version_string += *component_version;
         version_string += "\n";
       }
     } else {
-      comp_versions[*component_name] = *component_version;
-    }
-  }
-
-  // Format sorted component versions surrounded by (), wrapped
-  // e.g.
-  // (ares: 1.10.1-DEV, http_parser: 2.7.0, icu: 57.1, modules: 48,
-  //  openssl: 1.0.2j, uv: 1.9.1, v8: 5.1.281.84, zlib: 1.2.8)
-  const size_t wrap = 80;
-  version_string += "(";
-  const char* separator = "";
-  std::string versions = "";
-  for (auto it : comp_versions) {
-    std::string comp_version_string = it.first;
-    comp_version_string += ": ";
-    comp_version_string += it.second;
-    versions += separator;
-    if (wrap - (versions.length() % wrap) < comp_version_string.length()) {
-      versions += "\n ";
+      // Other component versions follow, comma separated, wrapped at 80 characters
+      std::string comp_version_string = *component_name;
+      comp_version_string += ": ";
+      comp_version_string += *component_version;
+      if (wrap == 0) {
+        wrap = comp_version_string.length();
+      } else {
+        wrap += comp_version_string.length() + 2;  // includes separator
+        if (wrap > 80) {
+          comp_versions += "\n ";
```

Comment:
The last component before the wrap doesn't get a `,` appended. Personally it doesn't bother me enough to block this PR.

Reply:
Well spotted, fixed, thanks